src/@types/assets/window.d.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies.
+ */
+import { _Hooks } from '@wordpress/hooks/build-types/createHooks';
+
+export {};
+
+declare global {
+	interface Window {
+		// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		wp: any;
```

Comment:
Would be nice to have a real type here, but I guess Gutenberg does this too :/
https://github.com/WordPress/gutenberg/blob/2c570001ff9dda80b588989accdec221bb8fcc6b/packages/e2e-test-utils-playwright/src/types.ts

Reply:
Any idea how can I get the real type for this?

Reply:
Not sure, exploring in the gutenberg repo would be your best bet. But the `wp` global has been around for a long time, so maybe it is untyped.

Reply:
I did explored it via Search Engine but found nothing.